kitsune/search/v2/documents.py

```diff
@@ -15,30 +19,41 @@
 
 
 class WikiDocument(SumoDocument):
-    url = field.Keyword()
-    updated = field.Date()
+    """ES Document for Knowledge Base.
+
+    Each document in ES is built on top of a parent KB article.
+    All the children are included in the same document.
+    """
 
+    updated = field.Date()
     product_ids = field.Keyword(multi=True)
     topic_ids = field.Keyword(multi=True)
+    locale = field.Keyword()
+    category = field.Keyword()
+    recent_helpful_votes = field.Integer()
 
     # Document specific fields (locale aware)
+    url = SumoLocaleAwareKeywordField(store=True)
+    slug = SumoLocaleAwareKeywordField(store=True)
     title = SumoLocaleAwareTextField()
     content = SumoLocaleAwareTextField(store=True, term_vector="with_positions_offsets")
     summary = SumoLocaleAwareTextField(store=True, term_vector="with_positions_offsets")
-    keywords = SumoLocaleAwareTextField(multi=True)
-
-    locale = field.Keyword()
-    current_id = field.Integer()
-    parent_id = field.Integer()
-    category = field.Keyword()
-    slug = field.Keyword()
-    is_archived = field.Boolean()
-    recent_helpful_votes = field.Integer()
+    keywords = SumoLocaleAwareKeywordField()
```

Comment:
This should either be a LocaleAwareTextField, or we should set `multi=True` here and split the string of keywords at whitespace in a `prepare_keywords` method